### ISSUE TITLE
MCSL reward problem.

### ISSUE THREAD

**Issue by Joonez** [BenCodez/VotingPlugin#737]
**Versions**
VotingPlugin: 5.20.8 'git-Paper-"d63075d" (MC: 1.15.2)'

**Relevant information**
25.03 21:52:19 [Server] WARN VoteSite with service site 'MCSL' does not exist, attempting to generaterate...
25.03 21:52:19 [Server] WARN VoteSite MCSL does not exist, creating one, set AutoCreateVoteSites to false to prevent this

**Context**

So I changed the MCSL part in the VoteSite to minecraft-server-list.com. The voting works fine but the reward is not working for some reason. Any ideas?

**Comment by BenCodez** [comment]
The service site is MSCL

**Comment by Joonez** [comment]
> The service site is MSCL
> […](#)

I know sorry figured that out after I posted but hey does the reward for that not work. It’s the same like others.

**Comment by BenCodez** [comment]
VoteSites.yml?

**Comment by Joonez** [comment]
#
# Reward for any site
# Similar to AnySiteReward
# This requires the site to be configured in order to get the reward.
VoteSites:
  Minecraft_erver_list_com:
    Enabled: true
    Name: Minecraft-server-list.com
    Priority: 1
    ServiceSite: Minecraft-server-list.com
    VoteURL: https://minecraft-server-list.com/server/454538/vote/
    VoteDelay: 24
    WaitUntilVoteDelay: false
    VoteDelayDaily: false
    ForceOffline: false
    Item:
      Material: DIAMOND
      Amount: 1
    Rewards:
      Commands:
        Console:
        - give %player% emerald 1
        - give %player% gold_ingot 3
        - give %player% experience_bottle 15
      Messages:
        Player: '&aThanks for voting on %ServiceSite%!'

  PlanetMinecraft_com:
    Enabled: true
    Name: PlanetMinecraft.com
    Priority: 3
    ServiceSite: PlanetMinecraft.com
    VoteURL: https://www.planetminecraft.com/server/oldbutgold/vote/
    VoteDelay: 24
    WaitUntilVoteDelay: false
    VoteDelayDaily: false
    ForceOffline: false
    Item:
      Material: DIAMOND
      Amount: 1
    Rewards:
      Commands:
        Console:
        - give %player% emerald 1
        - give %player% diamond 1
        - give %player% gold_ingot 3
        - give %player% experience_bottle 15
      Messages:
        Player: '&aThanks for voting on %ServiceSite%!'

  MinecraftServer_com:

    Enabled: true
    Name: MinecraftServer.org
    Priority: 5
    ServiceSite: minecraftservers.org
    VoteURL: https://minecraftservers.org/vote/576746
    VoteDelay: 24
    WaitUntilVoteDelay: false
    VoteDelayDaily: false
    ForceOffline: false
    Item:
      Material: DIAMOND
      Amount: 1
    Rewards:
      Commands:
        Console:
        - give %player% emerald 1
        - give %player% ender_pearl 1
        - give %player% experience_bottle 15
      Messages:
        Player: '&aThanks for voting on %ServiceSite%!'



EverySiteReward: []

Am Do., 26. März 2020 um 16:18 Uhr schrieb Ben <notifications@github.com>:

**Comment by BenCodez** [comment]
Pastebin it.

**Comment by Joonez** [comment]
https://pastebin.com/tx7gzs17

Am Do., 26. März 2020 um 16:25 Uhr schrieb Ben <notifications@github.com>:

**Comment by BenCodez** [comment]
ServiceSite: Minecraft-server-list.com

Change the service site to MSCL

**Comment by Joonez** [comment]
I tried it again. And this happened.
Error:
26.03 20:29:34 [Server] INFO Received a vote from service site 'MCSL' by
player 'Joonez'!
26.03 20:29:34 [Server] WARN VoteSite with service site 'MCSL' does not
exist, attempting to generaterate...
26.03 20:29:34 [Server] WARN VoteSite MCSL does not exist, creating one,
set AutoCreateVoteSites to false to prevent this

And I changed it to MCSL and restarted the server. But still no rewards for
MCSL. Only the other two.

Am Do., 26. März 2020 um 16:48 Uhr schrieb Ben <notifications@github.com>:
